src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -53,11 +53,22 @@ def get_cookie_value(self, name):
         raise ValueError("Cookie with name %s not found." % name)
 
     @keyword
-    def add_cookie(self, name, value, path=None, domain=None, secure=None):
+    def get_cookie_expiry(self, name):
```

Comment:
I don't exactly right now remember what the Selenium `get_cookie` returns, but if it's a dictionary (like it looks like) it might be feasible to write a keyword which return the whole cookie object. Or write a keyword which will return specific key from the cookie. Example like:
```
| Get Cookie | name |

Get Cookie Value | name | key |
```

The later example is enhancing existing keyword and therefore it would be good keep the backwards compatible as much as possible. There by default it should return the cookie value key.